File: markov/src/ts/getCSV.ts
import * as csv from 'csv-parser';
import * as fs from 'fs';

const data: any[] = [];

// CSVファイルを開く
fs.createReadStream('/path/to/your/file.csv')
  .pipe(csv())
  .on('data', (row: any) => {
    data.push(row);
  })
  .on('end', () => {
    // 最初の行を表示
    console.log(data[0]);
  });

File: markov/src/python/getCSV.py
import csv

data = []

# CSVファイルを開く
with open('/path/to/your/file.csv', 'r') as file:
    reader = csv.reader(file)

    for row in reader:
        data.append(row)

# 最初の行を表示
print(data[0])
